src/util.h

```diff
@@ -645,7 +645,7 @@ inline QString svrRegStatusToString ( ESvrRegStatus eSvrRegStatus )
         return QCoreApplication::translate ( "CServerDlg", "Registered" );
 
     case SRS_SERVER_LIST_FULL:
-        return QCoreApplication::translate ( "CServerDlg", "Directory Server full" );
+        return QCoreApplication::translate ( "CServerDlg", "Directory server list full" );
```

Comment:
I'm not an english expert, but, isn't it `Server's list directory full` instead?

Reply:
No, that's wrong.  It's the Directory that holds a server list.  That's the point of a Directory.